src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const pictures = document.querySelectorAll('.list-item__link');
+const valide = document.getElementById('largeImg');
+
+for (const picture of pictures) {
```

Comment:
Use the forEach method instead of the traditional for...of loop for iterating over the 'pictures' array.